index.html

```diff
@@ -222,7 +261,6 @@ <h4>mordecellon@gmail.com</h4>
       <div class="contact-bar">
         <img class="end-line" src="media/Shape.png" alt=" end line icon" />
       </div>
```

Comment:
Please make sure to eliminate white spaces on your website. There is a white space underneath your footer icons.  

> Your design

![spaces](https://user-images.githubusercontent.com/71140133/200094062-9b707b17-45e1-4af0-a81d-eaaf97f7de1b.PNG)